docs/client-connectors/databricks-client/index.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 1
+title: Databricks Client Connector
+---
+
+The Macrometa client connector for [Databricks](https://www.databricks.com/) enables you to seamlessly connect Macrometa [streams](../../streams/) and [collections](../../collections/) with Apache Spark within the Databricks environment. This connector allows you to leverage Spark's advanced processing capabilities, enabling you to derive valuable insights and make data-driven decisions.
+
+The Macrometa Spark Connector offers two main components: Stream Data Connector and Collection Data Connector.
+
+## Stream Data Connector
+
+The Stream Data Connector handles reading from and writing to real-time data streams.
+
+Source and target operations for streams can be executed using the format `com.macrometa.spark.stream.MacrometaTableProvider`.
+
+## Collection Data Connector
+
+The Collection Data Connector focuses on batch data processing to and from Macrometa collections.
+
+Source and target operations for collections can be executed using the format `com.macrometa.spark.collection.MacrometaTableProvider`.
+
+## Mapping Considerations
+
+When mapping from a Macrometa array to a Spark array, Macrometa uses ArrayType. ArrayType which is a collection data type that extends the DataType class, which is a superclass of all types in PySpark. All elements of ArrayType should have the same type of elements.
```

Comment:
```suggestion
When mapping from a Macrometa array to a Spark array, Macrometa uses ArrayType. ArrayType which is a collection data type that extends the DataType class, which is a superclass of all types in Spark. All elements of ArrayType should have the same type of elements.
```

Reply:
Just one comment from my end, I missed to change this in original doc it seems.. we don't need to say PySpark, we can say its a superclass of all types in Spark in general.